### DISCUSSION TITLE
Oppening repositories error/infinite loading

### DISCUSSION THREAD

**Discussion by william100806** [community/community#106677] [Repositories]
### Select Topic Area

Bug

### Body

Upon trying to open a repository I am met with "Error loading page It looks like your internet connection is down. Please check it." If I reload my page it will work normally, but only for that page, if I try to open a different section the error will pop up again. The second part is that if I click on my profile picture there are several links that will display "loading..." forever

**Comment by Olayiwolaaa** [comment]
Probably your network, but open another browser and try (OR incognito mode)

**Reply by william100806** [reply]
Opening on incognito worked as well as another browser worked but google didnt

**Reply by Olayiwolaaa** [reply]
Go to settings and try clearing data or cache of github. That should clear all data of github on your chrome browser.

# Please mark as answer if it works

**Reply by william100806** [reply]
I cleared all cache on github and still didn't work

**Reply by Olayiwolaaa** [reply]
Can i get a screenshot?

**Reply by rudolf61** [reply]
I have the exact same problem. I can't open files from Chrome. It does work when I use FireFox.

![Gitlab error loading any file within Chrome](https://github.com/community/community/assets/5022181/d40cab40-6f75-4eeb-a2d1-7115e822c413)

**Reply by rudolf61** [reply]
This problem is caused by Chrome extension. I was triggered when I saw "content blocked" in the console. I removed most of my extensions.

**Reply by william100806** [reply]
sorry I for got to answer, a screenshot of what?

**Reply by rudolf61** [reply]
Error loading page. It looks like your internet connection is down. This is a weird error message, since the internet connection is not down. When I took a look at the console, I saw the message that the content of Github is blocked. So I removed the two extensions (AdBlock and another one. I can't recall) and everything works again.
